src/app/visual/date-pipe/date-pipe-visual.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="app-screenshot">
+  <h3>
+    Standard implementation
+  </h3>
+
+  <table class="sky-table">
```

Comment:
Got a couple of HTML attributes here that need to be on new lines.

Reply:
Done.